observability/dashboards/all/logging_application.yml

```diff
@@ -185,11 +236,15 @@ spec:
             "type": "query"
           },
           {
-            "allValue": ".+",
+            "allValue": ".*",
```

Comment:
warum *?

Reply:
Der + Character wird in der URL ja schon verwendet, um Get-Parameter zu trennen. Ich brauch das als Stern, damit im Explore Tab auch wirklich alles bei All ausgewählt wird. Es hat ansonsten auf das Dashboard keinen Einfluss.